docs/source.rst

```diff
@@ -11,6 +11,14 @@ Source Guide
    Leaks With the Press <https://freedom.press/news/sharing-sensitive-leaks-press/>`__.
 
 
+.. warning:: Freedom of the Press Foundation has no access to any other 
+   organization's SecureDrop instance, and cannot assist directly in your communications
+   with them. We welcome general communications about SecureDrop, but if you
+   plan to use it to maintain your anonymity, it's important that you
+   do not discuss your own use of it with others via unsafe methods, including
+   email.
+
```

Comment:
I would word this even more specifically:

```
warning:: If you plan to use SecureDrop to maintain your anonymity, be aware that contacting 
    Freedom of the Press Foundation for technical support, in particular using your real name, 
    email address, or identifying details, can compromise this anonymity.
```

Reply:
Agreed but I don't like implying that there are situations where sources should contact FPF for support with other orgs' instances. Updated to ++specificity.

Reply:
lgtm